src/utils/server/pages/props.ts

```diff
@@ -66,7 +69,14 @@ export async function getProjectProps(
 
     const cv = isV3ProjectResult ? CV_V3 : CV_V2
 
-    return { props: { metadata, projectId, cv } }
+    return {
+      props: {
+        metadata,
+        projectId,
+        cv,
+      },
+      revalidate: 10, // 10 seconds https://nextjs.org/docs/api-reference/data-fetching/get-static-props#revalidate
```

Comment:
This is the only change worth caring about in this PR

Reply:
only thing I worry about this is blowing up our vercel edge cost